website/server/libs/api-v3/email.js

```diff
@@ -64,6 +65,19 @@ export function getUserInfo (user, fields = []) {
   return info;
 }
 
+export function getGroupUrl (group) {
+  let groupUrl;
+  if (group._id === TAVERN_ID) {
+    groupUrl = '/#/options/groups/tavern';
+  } else if (group.type === 'guild') {
+    groupUrl = `/#/options/groups/guilds/${group._id}`;
+  } else {
```

Comment:
Let's make this `else  if (group.type === 'party') {`, just to be clear about it.